woodwork/statistics_utils/_convert_ordinal_to_numeric.py

```diff
@@ -0,0 +1,16 @@
+def _convert_ordinal_to_numeric(schema, data):
+    """Transforms ordinal data to numeric data so that dependence can be calculated.
+
+    Args:
+        schema (woodwork.TableSchema): Woodwork typing info for the data.
+        data (dict[pd.Series]): Dictionary of Pandas series to use for
+            calculating dependence.
+
+    Returns:
+        None
+    """
+    for col_name in data.keys():
```

Comment:
there's no way to know this without looping through all the columns?

Reply:
I think this would be the most clean and understandable way to do this. We could use `filter` logic, but I believe we would still need to loop and it could make the code less readable.